### ISSUE TITLE
More Info Screen - VoiceOver Does Not read Combo FAQs PDF Headings 

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#80]
Page/PDF's hard to read. VoiceOver is not reading any of what you would think are headings in the document. Plus, the text is small and difficult to read.

**Comment by carolynhuang** [comment]
This cannot be resolved. As discussed, the PDFs do not have OCR, so VoiceOver will not read PDF.

**Comment by jfc3-dol** [comment]
I don't believe this PDF has been run through OCR software to change the image into text. I think the problem is because iOS and Android don't read any of the structure created for the PDFs such as headings, tables, etc. or this document is not tagged and on top of the PDF reading issue.

**Comment by jfc3-dol** [comment]
While looking at GitHub iOS item #80 "More Info Screen - VoiceOver Does Not read Combo FAQs PDF Headings" issue. VoiceOver read the document without any of the structure (heading, image ALt text, etc.) that should be in the document. When I found the same document on DOL.gov in the ILAB reports/Sweat and Toil section, I downloaded it. JAWS did not read any of the text because it wasn't tagged/created properly.

I also checked a few other documents such as the TDA Book from 2016, India 2016 information, both English and Spanish versions of Argentina 2016 and all of these were not tagged properly either which means JAWS could not read them. JAWS would read "Alter colon document not available". You need to look into all your PDFs because the ones I checked at least were not accessible.

There is still an issue with iOS reading properly structured and tagged PDFs.

**Comment by clguidi** [comment]
Thank you John, I will communicate this to the team responsible.

**Comment by jfc3-dol** [comment]
@clguidi,

I need to leave it open, so I remember to check each time the application is updated because it's an accessibility issue I found while testing.

**Comment by clguidi** [comment]
Hi @jfc3-dol, we no longer include this PDF in the app. Could you please close it when you have a moment?

**Comment by jfc3-dol** [comment]
@clguidi 

I only close the issues after I have tested them and made sure they were corrected. So I will need to check if this is the case with the next version of the application.

**Comment by clguidi** [comment]
@jfc3-dol I understand that, but this is a reminder to please check this issue and close it as we no longer include this PDF in the app. Thank you.

**Comment by jfc3-dol** [comment]
@clguidi 

Yes, the "Combo FAQs" PDF is no longer there, but the "Foreword" and "Sweat and Toil Magazine" seem to be PDFs and are not being read correctly because of the way the iOS platform reads PDFs. The same applies to Android too.

So I'm going to keep this open more to remind us to check it now and again to see if it's been fixed or not.

**Comment by clguidi** [comment]
@jfc3-dol Just FYI, we are removing these PDFs this year. The app developers will be releasing the final build without these PDFs. Please check this issue and close it once you have verified that the PDFs are no longer there. Thank you!

**Comment by jfc3-dol** [comment]
@pbhatt17 

Yes, those PDFs have been removed, so I'm going to close this issue.
